resources/views/users/client/home.blade.php

```diff
@@ -56,7 +56,7 @@
           </div>
         </div>
       </div>
-
+      {{-- {{ $projects->onEachSide(5)->links() }} --}}
```

Comment:
Please delete all the commented code on your files

Reply:
this code comment will have their importance